### ISSUE TITLE
help: a similar name exists in the module: `AtomicU8`

### ISSUE THREAD

**Issue by ykxVK8yL5L** [tokio-rs/axum#643]
### Version
axum = "0.4.2"
### Platform
- armv5te-unknown-linux-musleabi
- mips-unknown-linux-musl
- mipsel-unknown-linux-musl
### Crates
axum-0.4.2/src/routing/mod.rs:55:33

### Description
```
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/axum-0.4.2/src/routing/mod.rs:55:33
   |
55 |         use std::sync::atomic::{AtomicU64, Ordering};
   |                                 ^^^^^^^^^
   |                                 |
   |                                 no `AtomicU64` in `sync::atomic`
   |                                 help: a similar name exists in the module: `AtomicU8`
```

**Comment by davidpdrsn** [comment]
Probably a duplicate of https://github.com/tokio-rs/axum/issues/596. It has been fixed but not released yet.

Does using `main` via a git dependency work for you?

**Comment by ykxVK8yL5L** [comment]
> 
Thank for your reply.
When will it be released?

**Comment by davidpdrsn** [comment]
Sometime this week

**Comment by davidpdrsn** [comment]
It seems your problem was fixed so I’ll close this. Just re-open if I’m mistaken.

**Comment by ykxVK8yL5L** [comment]
> It seems your problem was fixed so I’ll close this. Just re-open if I’m mistaken.

After changing the dependencies to git .
The problem sloved. Thank you again.

**Comment by davidpdrsn** [comment]
Its published in axum 0.4.3. Enjoy!

**Comment by xdrghbjiikmdghu** [comment]
> Its published in axum 0.4.3. Enjoy!

Awesome.  Thank you very much  👍😀
